src/components/designSystem/Table/Table.tsx

```diff
@@ -92,6 +93,225 @@ const countMaxSpaceColumns = <T,>(columns: TableColumn<T>[]) =>
     return acc
   }, 0)
 
+const TableRow = ({
+  children,
+  className,
+  isClickable,
+  ...props
+}: TableRowProps & {
+  isClickable?: boolean
+}) => {
+  return (
+    <MUITableRow
+      className={tw(
+        {
+          'cursor-pointer': !!isClickable,
+        },
+        className,
+      )}
+      sx={{
+        '&:hover:not(:active), &:focus:not(:active), &:hover:not(:active) .lago-table-action-cell, &:focus:not(:active) .lago-table-action-cell':
+          {
+            backgroundColor: isClickable ? theme.palette.grey[100] : undefined,
+          },
+        '&:active, &:active .lago-table-action-cell': {
+          backgroundColor: isClickable ? theme.palette.grey[200] : undefined,
+        },
+        // Remove hover effect when action column is hovered
+        '&:has([data-id="actionColumn"] button:hover)': {
+          backgroundColor: 'unset !important',
+
+          '& .lago-table-action-cell': {
+            backgroundColor: theme.palette.background.paper,
+          },
+        },
+      }}
+      {...props}
+    >
+      {children}
+    </MUITableRow>
+  )
+}
+
+const TableCell = ({
+  children,
+  className,
+  hasPlaceholderDisplayed,
+  isBlurred,
+  maxSpace,
+  ...props
+}: PropsWithChildren &
+  TableCellProps & {
+    className?: string
+    isBlurred?: boolean
+    hasPlaceholderDisplayed?: boolean
+    maxSpace?: number
+  }) => {
+  return (
+    <MUITableCell
+      className={tw('lago-table-cell w-auto whitespace-nowrap p-0', className)}
```

Comment:
WDYT about separating classnames like this to avoid mixing tailwind classes and custom ones?

```suggestion
      className={tw('lago-table-cell', 'w-auto whitespace-nowrap p-0', className)}
```

Reply:
that's a good point, love it
updating